src/apps/frontend/pages/authentication/phone-login/country-code-select.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import COUNTRIES from '../../../constants/countries';
+
+interface CountryCodeSelectProps {
+  isLoading: boolean;
+  value: [string, string];
+  handleChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+}
+
+const CountryCodeSelect: React.FC<CountryCodeSelectProps> = ({
+  isLoading,
+  value,
+  handleChange,
+}) => (
+  <select
```

Comment:
better to create a separate component for this (if the existing dropdown doesn't work) and use it here.

Reply:
However, the problem is with the options data. The options data for countries will differ if we create a separate component of this

Reply:
We should be able to create a select component with look and feel we want and pass select option as child elements